zeek-config.h.in

```diff
@@ -298,3 +298,9 @@
 #else
 	#define ZEEK_DISABLE_TSAN
 #endif
+
+/* compiled with Spicy support */
+#cmakedefine HAVE_SPICY
+
+// Version of Spicy that we are compiling against.
+#define SPICY_VERSION_NUMBER ${SPICY_VERSION_NUMBER}
```

Comment:
> Couldn't `src/spicy` use `include hilti/runtime/include/version.h` directly? Or otherwise a new `spicy-version.h` just down in `src/spicy`?

Ack, that makes sense. (Except that Spicy's `version.h` calls it `PROJECT_VERSION_NUMBER`, which isn't great; but I can define a local alias).